static/src/javascripts/projects/common/modules/experiments/ab-ophan.ts

```diff
@@ -4,10 +4,9 @@ import config from '../../../../lib/config';
 import { noop } from '../../../../lib/noop';
 import reportError from '../../../../lib/report-error';
 
-// eslint-disable-next-line @typescript-eslint/no-explicit-any -- generics don’t play nice
-type BooleanFunction = (...args: any[]) => boolean;
+type BooleanFunction<Args extends unknown> = (...args: Args[]) => boolean;
```

Comment:
Why `extends unknown`?

Reply:
It was in fact necessary to write extend `unknown[]` for the generic of `not` to work.